conf/application.conf

```diff
@@ -144,16 +156,28 @@ microservice {
     claim {
         host = localhost
         port = 7502
+        context-base = "/tpi/postoverpaymentclaim/v1"
     }
 
     declaration{
         host = localhost
         port = 7502
+        context-base = "/accounts/overpaymentdeclarationdisplay/v1"
     }
+
+    fileupload {
+        host = localhost
+        port = 7502
+        context-base = "/filetransfer/init/1.0.0"
```

Comment:
This end point is incorrect. The `1.0.0` should be `v1`. It doesn't even match the endpoint you defined in the stubs. Have you tested this with the stub simulating DEC-64 responses?

Reply:
The pdf has this as the enpoint. Richard will get back to me about it